quickjs.h

```diff
@@ -705,6 +705,7 @@ JS_EXTERN int JS_SetPrototype(JSContext *ctx, JSValue obj, JSValue proto_val);
 JS_EXTERN JSValue JS_GetPrototype(JSContext *ctx, JSValue val);
 JS_EXTERN int JS_GetLength(JSContext *ctx, JSValue obj, int64_t *pres);
 JS_EXTERN int JS_SetLength(JSContext *ctx, JSValue obj, int64_t len);
+JS_EXTERN JSValue JS_ObjectSeal(JSContext *ctx, JSValue obj, JS_BOOL freeze);
```

Comment:
Sorry for circling back to this but, on second thought, it's probably better to add two API functions, JS_ObjectFreeze and JS_ObjectSeal, and drop the boolean.

Reply:
Well, we expose a C API so `JS_FreezeObject`, `JS_SealObject` are better imo.